lib/dgram.js

```diff
@@ -121,6 +122,12 @@ function Socket(type, listener) {
 
   // If true - UV_UDP_REUSEADDR flag will be set
   this._reuseAddr = options && options.reuseAddr;
+  
+  if (options && options.recvBufferSize)
```

Comment:
If we're adding more options, it might be better to just make sure `options` is an object. Adding these underscored properties also means we have to maintain them indefinitely.